backend/src/main/scala/cromwell/backend/validation/ValidatedRuntimeAttributesBuilder.scala

```diff
@@ -64,7 +64,7 @@ trait ValidatedRuntimeAttributesBuilder {
       case Invalid(nel) => throw new RuntimeException with MessageAggregation with NoStackTrace {
         override def exceptionContext: String = "Runtime attribute validation failed"
 
-        override def errorMessages: Traversable[String] = nel.toList
+        override def errorMessages: Iterable[String] = nel.toList
```

Comment:
this should be a `List`? especialy if it's from `nel.toList`

Reply:
again just trying to stick close to the original